File: Stack/stack.js
let Stack = function( ){
	this.count = 0;
	this.storage = {};

	this.push = function( value ) {
		this.storage[this.count] = value;
		this.count++;
	}

	this.pop = function(){
		if( this.count === 0)
			return undefined
		delete this.storage[this.count-1];
		this.count--;
		return this.storage;
	}

	this.size = function(){
		return this.count;
	}

	this.peek = function(){
		return this.storage[this.count-1];
	}
}

var s1 = new Stack();
s1.push('google.com');
s1.push('facebook.com');
s1.push('github.com');
console.log(s1.storage);
console.log(s1.peek());

File: Stack/Stack.py
class Stack:
    def __init__(self):
        self.size = 0
        self.list = []
    def isEmpty(self):
        return self.size == 0
    def push(self, data):
        self.list.append(data)
        self.size+=1
    def pop(self):
        self.list.pop()
        self.size-=1
    def peek(self):
        return self.list[len(self.list)-1]

stck = Stack()
stck.push(10)
stck.push(35)
stck.push(100)
stck.pop()
print stck.size
print stck.list
print stck.peek()
print stck.isEmpty()

